apps/storybook-react/.storybook/manager.ts

```diff
@@ -1,6 +1,6 @@
 import { addons } from '@storybook/manager-api';
-import MetaMaskTheme from './theme';
+import { themes } from '@storybook/theming';
 
 addons.setConfig({
-  theme: MetaMaskTheme,
+  theme: themes.light,
 });
```

Comment:
Should be removed after rebased from https://github.com/MetaMask/metamask-design-system/pull/608